LICENSE

```diff
@@ -225,3 +225,4 @@ The text of each license is the standard Apache 2.0 license.
    mvnw files from https://github.com/apache/maven-wrapper Apache 2.0
    svg files from skywalking-ui/src/assets/icons: https://github.com/google/material-design-icons Apache 2.0
    ZipkinQueryHandler.java reference from zipkin2.server.internal.ZipkinQueryApiV2: https://github.com/openzipkin/zipkin  Apache 2.0
+   async-profiler parser files in server-library/library-async-profiler-jfr-parser from https://github.com/open-telemetry/opentelemetry-proto/tree/v0.7.0 Apache 2.0
```

Comment:
You seem to copy `from ...` from somewhere, please fix it with the correct source of these codes.